libc/startup/linux/x86_64/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ add_startup_object(
     libc.src.string.memory_utils.inline_memcpy
     libc.src.unistd.environ
   COMPILE_OPTIONS
+    -fno-stack-protector
```

Comment:
should this also be added to the riscv startup options?

Reply:
Waiting for specifications from @nickdesaulniers